ballerina/api_inspectionlot_srv/docs.json

```
@@ -0,0 +1,7 @@
+{
+  "description": "The `ballerinax/sap.s4hana.api_inspectionlot_srv` package offers APIs for seamless integration with the [Inspection Lot API v1.2.0](https://api.sap.com/api/API_INFORECORD_PROCESS_SRV/overview).In the SAP system, you process quality inspections on the basis of inspection lots. The data relating to a quality inspection is grouped into several entities. With this service it is possible, for example, to update inspection lot header data or to create inspection results and usage decisions.",
+  "communication-scenario": "Quality Inspection Process Integration",
+  "import-statement": "import ballerinax/sap.s4hana.api_inspectionlot_srv as inspectionlot",
+  "client-init": "inspectionlot:Client inspectionlotClient = check new (",
+  "api-invocation": "inspectionlot:A_InspectionLotWrapper|error createdInspectionlot = check inspectionlotClient->createA_Inspectionlot();"
```

```suggestion
  "api-invocation": "inspectionlot:A_InspectionLotWrapper createdInspectionlot = check inspectionlotClient->createA_Inspectionlot();"
```
No need for error since check is there

---

Updated here [Update docs.json,settings.gradle and build.gradle for PR build](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/11/commits/187ec88523c7b17afc72a2961fecad9d88601941)